core/src/main/java/org/apache/iceberg/BaseTransaction.java

```diff
@@ -454,6 +454,7 @@ public TableMetadata refresh() {
     }
 
     @Override
+    @SuppressWarnings("ConsistentOverrides")
     public void commit(TableMetadata underlyingBase, TableMetadata metadata) {
```

Comment:
renaming `underlyingBase` to `base` here does conflict and shadows the `base` variable that's defined in the class itself